server/events.lua

```diff
@@ -1,14 +1,31 @@
 -- Event Handler
 
+local usedLicenses = {}
+
 AddEventHandler('chatMessage', function(_, _, message)
     if string.sub(message, 1, 1) == '/' then
         CancelEvent()
         return
     end
 end)
 
+AddEventHandler('playerJoining', function()
+    if not QBConfig.Server.CheckDuplicateLicense then return end
+    local src = source
+    local license = QBCore.Functions.GetIdentifier(src, 'license')
+    if not license then return end
```

Comment:
If the player doesn't have a license, should they be dropped as well?

Reply:
No, this is already checked when the player connects, in the case that this is enabled in the config.